client/src/shared-components/navbar.css

```diff
@@ -0,0 +1,74 @@
+.navbar {
```

Comment:
(optional)
As a general principle, when we are relying on one CSS frameworks, React Bootstrap in this case, It's recommended to to use its features as much as possible.
So I can see some of the css properties could be replaced by bs class names. This makes your code cleaner (more readability) and make your custom css files fewer.
For example text-align: left with "text-start"
I know It's sometime tricky to consider all of them! It's up to you.